pom.xml

```diff
@@ -11,7 +11,7 @@
   <parent>
     <groupId>org.springframework.boot</groupId>
     <artifactId>spring-boot-starter-parent</artifactId>
-    <version>1.5.1.RELEASE</version>
+    <version>1.5.22.RELEASE</version>
```

Comment:
It can't be updated to 2.X version because it fails during the compilation

Reply:
It doesn't matters, I've seen it on wiki, it's ok.